queue/remote_unordered_test.go

```diff
@@ -74,16 +74,16 @@ func (s *RemoteUnorderedSuite) TearDownTest() {
 	s.cancel()
 }
 
-func (s *RemoteUnorderedSuite) TestDriverIsUnitializedByDefault() {
+func (s *remoteSuite) TestDriverIsUnitializedByDefault() {
```

Comment:
flagging `Unitialized`
(though it wasn't from this)

Reply:
Fixed.